plone/app/layout/viewlets/content.py

```diff
@@ -71,43 +71,7 @@ def show(self):
             ISecuritySchema,
             prefix='plone',
         )
-        return not self.anonymous or settings.allow_anon_views_about
```

Comment:
is just the way Git shows the change; look at this commit isolated: https://github.com/plone/plone.app.layout/pull/90/commits/1845be16f68cec3fea635032212259d8dac6de58